packages/apps/reputation-oracle/server/src/modules/escrow-completion/payouts-calculation/cvat-payouts-calculator.ts

```diff
@@ -76,4 +84,17 @@ export class CvatPayoutsCalculator implements EscrowPayoutsCalculator {
       }),
     );
   }
+
+  private parseJobBounty(jobBounty: string, tokenDecimals: bigint): bigint {
+    const parts = jobBounty.split('.');
+    if (parts.length > 1) {
+      const decimalsInBounty = parts[1].length;
+      if (decimalsInBounty > tokenDecimals) {
+        throw new Error(
```

Comment:
I don't think we need to throw an error here.
Wouldn't it be better to just remove excessive decimals?